src/Infrastructure/Infrastructure.csproj

```diff
@@ -17,4 +17,8 @@
     <ProjectReference Include="..\Application\Application.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Folder Include="Persistence\Migrations\" />
+  </ItemGroup>
+
```

Comment:
Why is this needed? My understanding is that all `.cs` files are automatically included by default.

Reply:
Since the folder is currently empty, vs added that